### ISSUE TITLE
Rename Records tab to Charts

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/ala-bie-hub#44]
**Background**
The 'Records' tab does not really display records, it displays collated record data categorised, organised, and presented in charts. Additionally, the word 'records' on the species pages tend to connect users to the Biocache. The Records tab does not have a Biocache style display. It is a bit misleading. 'Statistics' would be a more appropriate name.

**Requirements**
- Rename "Records" tab to "Statistics"

**Comment by javier-molina** [comment]
@adam-collins  sorry I know there is progress already on this but could we rename to tab to "Charts" to have some consistency with Biocache and address Ely's suggestion?

@aliciadoddy fyi.

**Comment by adam-collins** [comment]
pull request https://github.com/AtlasOfLivingAustralia/ala-bie-hub/pull/61

**Comment by aliciadoddy** [comment]
@javier-molina @adam-collins Had discussed with Ely since the meeting - even if 'statistics' is not technically accurate, it's more about what users might expect to find in a tab called 'statistics', which often gets thrown around as quite a broad term for figures, metrics, etc (even if that's not what statistics actually are). Happy to try another name, and I agree that 'Charts' would have more consistency with the Biocache as mentioned, but if we were to condense 'data providers' into the tab too (as discussed for consolidating tab content), then 'Charts' will not really represent the full info in that tab.

tldr - happy to go with 'Charts', but it still may not be a truly suitable name for this tab and its content.

**Comment by djtfmartin** [comment]
`metrics` is used on collectory pages already. 
https://collections.ala.org.au/public/show/dr376

its also in use by GBIF: https://www.gbif.org/dataset/4ce8e3f9-2546-4af1-b28d-e2eadf05dfd4/metrics

Consistency is probably key here.

**Comment by kylie-m** [comment]
UX team has discussed during standup today, decided on Charts to be consistent with Biocache (eg https://biocache.ala.org.au/occurrences/search?q=taxa%3A%22Macropus+giganteus%22#tab_chartsView)
